pkg/kubectl/cmd/cmd.go

```diff
@@ -110,23 +110,26 @@ __custom_func() {
 }
 `
 	valid_resources = `Valid resource types include:
-   * pods (aka 'po')
-   * replicationcontrollers (aka 'rc')
+   * componentstatuses (aka 'cs')
    * daemonsets (aka 'ds')
-   * services (aka 'svc')
+   * deployments
```

Comment:
Or you could use the discovery client and conditionally print them. Would be trickier probably (I am not familiar with it atm) but that is the right thing to do:)